app/perfunwind.cpp

```diff
@@ -506,6 +510,10 @@ void PerfUnwind::resolveCallchain()
     if (isKernel)
         return;
 
+    if (m_currentUnwind.sample->branchStack().size() > maxUnwindStack()) {
+        m_currentUnwind.isIncompleteCallchain = true;
```

Comment:
dito: when the LBR stack is fully exhausted but we don't reach fully reach the top we should find a way to set this to true also

Reply:
Probably it is the matter for future enhancement?

Reply:
yes, agreed. but please rephrase your commit message:

Track *if*  unwinding completed the full callchain or stopped earlier